xml/System.Workflow.Runtime/WorkflowRuntime.xml

```diff
@@ -1653,21 +1653,19 @@ static void Main(string[] args)
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- A workflow instance can be unloaded from memory by an explicit call to <xref:System.Workflow.Runtime.WorkflowInstance.Unload%2A>, or implicitly by the workflow run-time engine according to its own semantics. For example, the workflow run-time engine unloads a workflow instance if the instance becomes idle and the runtime has a <xref:System.Workflow.Runtime.Hosting.WorkflowPersistenceService> added for which <xref:System.Workflow.Runtime.Hosting.WorkflowPersistenceService.UnloadOnIdle%2A> is `true`.  
-  
- The workflow run-time engine raises the  
-  
- <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowUnloaded> event after the state of the workflow instance has been successfully persisted but before the instance is invalidated in memory. Therefore, a <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowPersisted> event precedes the <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowUnloaded> event.  
+## Remarks
+
+A workflow instance can be unloaded from memory by an explicit call to <xref:System.Workflow.Runtime.WorkflowInstance.Unload%2A>, or implicitly by the workflow run-time engine according to its own semantics. For example, the workflow run-time engine unloads a workflow instance if the instance becomes idle and the runtime has a <xref:System.Workflow.Runtime.Hosting.WorkflowPersistenceService> added for which <xref:System.Workflow.Runtime.Hosting.WorkflowPersistenceService.UnloadOnIdle%2A> is `true`.  
   
- For the <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowUnloaded> event, the sender contains the <xref:System.Workflow.Runtime.WorkflowRuntime> and <xref:System.Workflow.Runtime.WorkflowEventArgs> contains the <xref:System.Workflow.Runtime.WorkflowInstance> associated with the event.  
+The workflow run-time engine raises the `WorkflowUnloaded` event after the state of the workflow instance has been successfully persisted but before the instance is invalidated in memory. Therefore, a <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowPersisted> event precedes the <xref:System.Workflow.Runtime.WorkflowRuntime.WorkflowUnloaded> event.  
   
- For more information about handling events, see [Consuming Events](https://go.microsoft.com/fwlink/?LinkID=157352).  
+For the `WorkflowUnloaded` event, the sender contains the <xref:System.Workflow.Runtime.WorkflowRuntime> and <xref:System.Workflow.Runtime.WorkflowEventArgs> contains the <xref:System.Workflow.Runtime.WorkflowInstance> associated with the event.  
   
-   
+For more information about handling events, see [Consuming Events](https://go.microsoft.com/fwlink/?LinkID=157352).  
```

Comment:
not related to your PR, but this link goes to a very old topic. We do have an issue to review FwLinks. This just reinforces that.

Reply:
A good replacement is [Handling and raising events](~/docs/standard/events/index.md). I'll search for occurrences of **Consuming Events** and open a PR to replace them. I've replaced all 13 occurrences in this file.